packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -1785,6 +1785,7 @@ describe('shallow', () => {
         const wrapper = shallow(<MyComponent />, { disableLifecycleMethods: true });
         expect(wrapper.find(Table).length).to.equal(0);
         wrapper.instance().componentDidMount();
+        wrapper.update();
```

Comment:
the need for this addition concerns me; perhaps that's OK because most people won't be manually calling a lifecycle method, but this makes it seem like it might be a breaking change :-/

Reply:
finishing this up today

Reply:
i dont think we need this if we return true per your comment here: https://github.com/airbnb/enzyme/pull/2061#discussion_r267586179.  Checking though